web-ui/src/api/memberskill.js

```diff
@@ -25,11 +24,8 @@ export const getSelectedMemberSkills = async (id, cookie) => {
 
 export const getSkillMembers = async (id, cookie) => {
   return resolve({
-    url: memberSkillUrl,
+    url: `${memberSkillUrl}?skillid=${encodeURIComponent(id)}`,
     responseType: "json",
-    params: {
-      skillid: id,
-    },
```

Comment:
I'm curious why only this API function was changed to not use the `params` property. There are other API functions that still use `params`.

Reply:
This was a huge reminder, thank you! Params isn't a thing with fetch.